<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="BtnTimer">
      <property name="text">
       <string>显示时间</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="BtnRandomPos">
      <property name="text">
       <string>随机位置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newFile"/>
    <addaction name="save"/>
    <addaction name="saveAs"/>
    <addaction name="open"/>
    <addaction name="close"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_Z"/>
    <addaction name="action_C"/>
    <addaction name="actionFind"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newFile"/>
   <addaction name="open"/>
   <addaction name="save"/>
   <addaction name="saveAs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newFile">
   <property name="icon">
    <iconset resource="myicon.qrc">
     <normaloff>:/myImages/images/filenew.png</normaloff>:/myImages/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="statusTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="save">
   <property name="icon">
    <iconset resource="myicon.qrc">
     <normaloff>:/myImages/images/filesave.png</normaloff>:/myImages/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="icon">
    <iconset resource="myicon.qrc">
     <normaloff>:/myImages/images/filesaveas.png</normaloff>:/myImages/images/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="open">
   <property name="icon">
    <iconset resource="myicon.qrc">
     <normaloff>:/myImages/images/fileopen.png</normaloff>:/myImages/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="close">
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Z">
   <property name="text">
    <string>撤销(&amp;Z)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myicon.qrc"/>
 </resources>
 <connections/>
</ui>
